src/main/java/com/friendly_machines/intellij/plugins/ideanative2debugger/impl/DebugProcess.java

```diff
@@ -553,33 +551,43 @@ public XDebuggerEditorsProvider getEditorsProvider() {
     }
     public void next(boolean reverse) throws GdbMiOperationException, IOException, InterruptedException {
-        gdbCall("-exec-next", reverse ? List.of("--reverse") : Collections.emptyList(), Collections.emptyList());
+        gdbCall("-exec-next", Collections.emptyList(), reverse ? List.of("reverse") : Collections.emptyList());
     }
     public void stepInstruction(boolean reverse) throws GdbMiOperationException, IOException, InterruptedException {
-        gdbCall("-exec-step-instruction", reverse ? List.of("--reverse") : Collections.emptyList(), Collections.emptyList());
+        gdbCall("-exec-step-instruction", Collections.emptyList(), reverse ? List.of("reverse") : Collections.emptyList());
     }
     public void nextInstruction(boolean reverse) throws GdbMiOperationException, IOException, InterruptedException {
-        gdbCall("-exec-next-instruction", reverse ? List.of("--reverse") : Collections.emptyList(), Collections.emptyList());
+        gdbCall("-exec-next-instruction", Collections.emptyList(), reverse ? List.of("reverse") : Collections.emptyList());
     }
 
     public void finish(boolean reverse) throws GdbMiOperationException, IOException, InterruptedException {
-        gdbCall("-exec-finish", reverse ? List.of("--reverse") : Collections.emptyList(), Collections.emptyList());
+        gdbCall("-exec-finish", Collections.emptyList(), reverse ? List.of("reverse") : Collections.emptyList());
     }
     public void until(Optional<String> location) throws GdbMiOperationException, IOException, InterruptedException {
-        gdbCall("-exec-until", location.map(List::of).orElse(Collections.emptyList()), Collections.emptyList());
+        gdbCall("-exec-until", location.map(List::of).orElse(Collections.emptyList()));
     }
     public void jump(String location) throws GdbMiOperationException, IOException, InterruptedException {
-        gdbCall("-exec-jump", List.of(location), Collections.emptyList());
+        gdbCall("-exec-jump", List.of(location));
     }
     public List<String> listFeatures() throws GdbMiOperationException, ClassCastException, IOException, InterruptedException {
         // For example, GDB 12.1 has ^done,features=["frozen-varobjs","pending-breakpoints","thread-info","data-read-memory-bytes","breakpoint-notifications","ada-task-info","language-option","info-gdb-mi-command","undefined-command-error-code","exec-run-start-option","data-disassemble-a-option","python"]
-        return (List<String>) gdbCall("-list-features", Collections.emptyList(), Collections.emptyList()).get("features");
+        return (List<String>) gdbCall("-list-features", Collections.emptyList()).get("features");
     }
-    public Object infoGdbMiCommand(String commandName) throws GdbMiOperationException, IOException, InterruptedException {
-        return gdbCall("-info-gdb-mi-command", List.of(commandName), Collections.emptyList()).get("command");
+
+    /**
+     * What's the expected return type, what is it used for?
```

Comment:
It's used in order to find out what commands the debugger supports.

Right now, there is only one field (`exists`) in the result, see https://sourceware.org/gdb/onlinedocs/gdb/GDB_002fMI-Support-Commands.html .